shell/platform/windows/window.cc

```diff
@@ -537,6 +537,9 @@ Window::HandleMessage(UINT const message,
       // DefWindowProc will send WM_CHAR for this WM_UNICHAR.
       break;
     }
+    case WM_THEMECHANGED:
+      OnThemeChange();
+      break;
```

Comment:
Could we add a unit test that `OnThemeChanged` is called when we send a `WM_THEMECHANGED` message?

Reply:
Do we have a method for manually sending messages to the window? Is this what `MockFlutterWindow::InjectWindowMessage` does?

Reply:
Yep - there are a few tests for text input event handling that do this, if I remember correctly -- check `window_unittests.cc`.

Reply:
Please check if the new `FlutterWindowTest.OnThemeChange` unit test fits what you had in mind.